### PR TITLE
chore: change datastage to catalyst

### PR DESCRIPTION
**Pull request by ZakirG**
to resolve https://ctds-planx.atlassian.net/browse/PXP-5016

the main files changing here are qa-dcp.planx-pla.net/portal/gitops.json and the images

the other json files are out of date copies of the gitops.json file

### CODE REVIEW

File: jenkins-dcp.planx-pla.net/portal/gitops.json [hunk 1]
```diff
@@ -31,18 +31,18 @@
     "projectDetails": "boardCounts"
   },
   "components": {
-    "appName": "NHLBI DataSTAGE",
+    "appName": "NHLBI BioData Catalyst",
     "index": {
       "introduction": {
-        "heading": "DataSTAGE",
-        "text": "DataSTAGE supports the management, analysis and sharing of human disease data for the research community and aims to advance basic understanding of the genetic basis of complex traits and accelerate discovery and development of therapies, diagnostic tests, and other technologies for diseases like cancer. The data commons supports cross-project analyses by harmonizing data from different projects through the collaborative development of a data dictionary, providing an API for data queries and download, and providing a cloud-based analysis workspace with rich tools and resources.",
+        "heading": "",
```
Comment (themarcelor) [inline jenkins-dcp.planx-pla.net/portal/gitops.json]: Do we really want to leave the heading empty?

Comment (ZakirG) [reply jenkins-dcp.planx-pla.net/portal/gitops.json]: the heading is already empty in production

Comment (ZakirG) [reply jenkins-dcp.planx-pla.net/portal/gitops.json]: i'm just updated the jenkins json to match

Comment (ZakirG) [reply jenkins-dcp.planx-pla.net/portal/gitops.json]: the main files changing here are qa-dcp.planx-pla.net/portal/gitops.json and the images

the other json files are out of date copies of the gitops.json file

we duplicate our config json in 4 places

Comment (themarcelor) [reply jenkins-dcp.planx-pla.net/portal/gitops.json]: Oh, ok. 👍

**Pull request merged**